recipes/mrtrix3/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mrtrix3" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MRtrix3/mrtrix3/archive/{{ version }}.tar.gz
+  sha256: 19652c21b1c259629cfee1af3e8cdf1c1ad625dad2f709c66bd6f1eeee9ce602
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+  host:
+    - eigen
+    - fftw
+    - libpng
+    - libtiff
+    - pkg-config
+    - zlib
+  run:
+    - fftw
+    - libpng
+    - libtiff
+    - zlib
```

Comment:
These shouldn't be needed but they get implicitly added via their `run_exports` with the correct constraints.
```suggestion
```